### DISCUSSION TITLE
Welcome to PhysioBrothers

### DISCUSSION THREAD

**Discussion by dhruvparmar2310** [community/community#31844] [Other Feature Feedback, Questions, & Ideas]
I wanna increase more view and visitor in my site...
If anyone know How? Please let me know...

You can visit my website link :
[Redacted due to Code of Conduct]

**Comment by melvincwng** [comment]
@dhruvparmar2310 Hey there, try sharing your website or project with your family, friends, classmates or even with online developers on various platforms like Medium.com.

You can also read up on SEO (Search Engine Optimization) techniques to make your website more searchable in search engines. This will help to boost traffic to your website.

Alternatively, you can contact digital marketing agencies to do the work for you.

Hope this answers your question, all the best 👍🏻

**Comment by jfullstackdev** [comment]
same thing I asked in the past when I started putting projects on my profile and they were very hard to be discovered, I even tried Google Analytics to speed up the traffic

one thing is for sure, you need to be consistent, active and make sure your content is reliable

I started following others both in GitHub and Twitter, when I started doing these things, I received so many stars on my repos

**Comment by queenofcorgis** [comment]
Hi @dhruvparmar2310 , we’re glad you’re here 😀
Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
